cmd/snap-confine/snap-confine.apparmor.in

```diff
@@ -22,7 +22,7 @@
     # the .so suffix, eg. ld-linux-aarch64.so.1
     /{,{,var/lib/snapd/}snap/{snapd,core}/*/}{,usr/}lib{,32,64,x32}/{,@{multiarch}/{,atomics/}}ld{-*,64}.so* mrix,
     # libc, you are funny
-    /{,{,var/lib/snapd/}snap/{snapd,core}/*/}{,usr/}lib{,32,64,x32}/{,@{multiarch}/{,atomics/}}libc{,-[0-9]*}.so* mr,
+    /{,{,var/lib/snapd/}snap/{snapd,core}/*/}{,usr/}lib{,32,64,x32}/{,glibc-hwcaps/{x86-64-v{2,3,4},power{9,10},z1{3,4,5,6}}/,@{multiarch}/{,atomics/}}libc{,-[0-9]*}.so* mr,
```

Comment:
Should this just say
```suggestion
    /{,{,var/lib/snapd/}snap/{snapd,core}/*/}{,usr/}lib{,32,64,x32}/{,glibc-hwcaps/*/,@{multiarch}/{,atomics/}}libc{,-[0-9]*}.so* mr,
```

Reply:
I agree this suggestion feels more maintainable

Reply:
@MingcongBai can you push a tweak?

Reply:
> @MingcongBai can you push a tweak?

Just pushed a new revision, thanks.